i18n/fr_FR/administration

```diff
@@ -0,0 +1,78 @@
+Configuration du serveur Apache pour EspoCRM 
+
+Ces instructions sont supplémentaires à la directive [Configuration du serveur] (server-configuration.md). Veuillez noter que 
+tous les paramètres de configuration affichées ici sont faits sur le serveur Ubuntu 
+
+PHP Exigences 
+Pour installer toutes les bibliothèques nécessaires, appliquez ces commandes dans un terminal 
+
+sudo apt-get update
+sudo apt-get install php-mysql php-json php-gd php-mcrypt php-zip php-imap php-mbstring php-curl
+sudo phpenmod mcrypt imap mbstring
+sudo service apache2 restart
+
+Résolution du problème "Erreur API:API EspoCRM est indisponible 
+Prenez seulement les étapes nécessaires.Après chaque étape, vérifiez si le problème est résolu. 
+
+1. Veuillez activer le support "mod_rewrite" dans Apache
+Pour activer "mod_rewrite", appliquez ces commandes dans un terminal
+
+sudo a2enmod rewrite
+sudo service apache2 restart
+
+2. Veuillez Activer le support .htaccess
+Pour l’activation du support .htaccess, ajoutez / modifiez les paramètres de la configuration du serveur/etc/apache2/sites-
+available/ESPO_VIRTUAL_HOST.conf or /etc/apache2/apache2.conf (/etc/httpd/conf/httpd.conf):
+
+<Directory /PATH_TO_ESPO/>
+AllowOverride All
+</Directory>
+
+Ensuite, demarrez cette commande dans un terminal:
+
+sudo service apache2 restart
+
+3. Ajouter le chemin RewriteBase
+Ouvrez un fichier /ESPOCRM_DIRECTORY/api/v1/.htaccess et remplacez la ligne suivante:
+
+# RewriteBase /
+
+Avec
+
+RewriteBase /REQUEST_URI/api/v1/
+où REQUEST_URI est une partie de l'URL e.g.. pour "http://example.com/espocrm/", REQUEST_URI est "espocrm"
+
+Veuillez activer le support HTTP AUTHORIZATION (seulement  pour FastCGI).
+FastCGI ne supporte pas HTTP AUTHORIZATION par défaut. Si vous utilisez FastCGI, vous devez l'activer dans votre VirtualHost
+ou /etc/apache2/apache2.conf (httpd.conf) en ajoutant le code suivant:
+
+Pour le module Fcgid
+
+<IfModule mod_fcgid.c>
+  FcgidPassHeader Authorization
+  FcgidPassHeader Proxy-Authorization
+  FcgidPassHeader HTTP_AUTHORIZATION  
+</IfModule>
+
+Pour le modüle FastCgi
+
+<IfModule mod_fastcgi.c>
+   FastCgiConfig -pass-header Authorization \
+                        -pass-header Proxy-Authorization \
+                        -pass-header HTTP_AUTHORIZATION  
+</IfModule>
+
+Pour vérifier quel module est actuellement utilisé, exécutez cette commande et trouvez le module: 
+
+Pour vérifier quel module est actuellement utilisé, exécutez cette commande et trouvez le module:     
+
+apache2ctl -M
+
+
+
+
+
+
+
+
+
```

Comment:
Sauvegarder et restauration
 
+Comment sauvegarder manuellement  EspoCRM 
 
EspoCRM se compose des fichiers et des données (data). Toutes ces données sont nécessaires pour sauvegarder complètement 'EspoCRM.Comment le faire. sur un serveur Ubuntu avec MySQL.Voici les instructions :
 
   Étape 1. Sauvegarder des fichiers
+
 
 
 
 ```bash
 tar -czf "files.tar.gz" -C /var/www/html .
 ```
+Créez une archive du contenu du répertoire d’EspoCRM installé. Pour Ubuntu,le chemin par défaut est / var / www / html. Vous pouvez appliquer cette commande.
+
+tar -czf "fichiers.tar.gz" -C / var / www / html.
 
 
+          Étape 2. Sauvegarder la base de données
+
 
 
 
 Pour sauvegarder toutes vos données, vous devez savoir  le nom de la base de données et l’accés des informations d'identification. Vous pouvez trouver le nom de la base de données dans le fichier de configuration /ESPOCRM_DIRECTORY/data/config.php sous la base de données de section. Vous pouvez utuliser cette commande pour sauvegarder votre base de données
+bash

+mysqldump --UTULİSATEUR = VOTRE_UTULİSATEUR --CODE DE CHİFFRE = VOTRE_CODE DE CHİFFRE VOTRE_BASE DE DONNES_NOM> "db.sql"
+
 ```
 
    Étape 3.Copier la sauvegarde    

+C'est tout. Maintenant, vous devez copier la sauvegarde créée dans une place sûr.
 

   Comment sauvegarder EspoCRM avec un script
 
 Vous pouvez utiliser un script pour sauvegarder toutes les données nécessaires. Connectez-vous via SSH et exécutez les commandes (testées sur le serveur Ubuntu
 
 
-```bash
+    Télécharger un scrip
+    
+    t```bash
 wget https://raw.githubusercontent.com/espocrm/documentation/master/_static/scripts/backup.sh
 ```
    Exécuter le script
 
 ```bash
 bash ./backup.sh PATH_TO_ESPOCRM BACKUP_PATH
 ```


+ 
+ où
  PATH_TO_ESPOCRM est un chemin d'accès au répertoire EspoCRM installé.
  BACKUP_PATH est un chemin d'accès au répertoire de sauvegarde
+
 
 For Ubuntu server it is:
+Pour le serveur Ubuntu,cela est :
 
 ```bash
 bash ./backup.sh /var/www/html /opt/backups
+bash ./backup.sh / var / www / html / opt / sauvegardes
 ```
 
 Note: If your MySQL user doesn't have needed rights to dump your database, you will be promted to enter credentials of another MySQL user.
 
+Remarquer: Si votre utilisateur MySQL n'a pas les droits nécessaires pour décharger  votre base de données, vous serez invité à entrer les informations d'identification d'un autre utilisateur MySQL.
+
 After successful creation, you will get a path to the created backup.
+Après une création réussie, vous allez obtenir un chemin vers la sauvegarde créée
+
 
 ## Restore EspoCRM from a backup
+  Restaurer EspoCRM à partir d'une sauvegarde
 
 You can restore EspoCRM from the backup created as described above.
+Vous pouvez restaurer EspoCRM à partir de la sauvegarde créée comme décrit ci-dessus.
 
 ### Step 1. Unarchive backup files
+    Étape 1. Désarchiver les fichiers de sauvegarde
 
 To unarchive files, you can use Archive Manager or run the below command. Files need to be placed in the web-server directory.
 
@@ -67,20 +103,42 @@ tar -xzf "files.tar.gz" -C /var/www/html
 ```
 where:
  * `/var/www/html` is a web-server directory.
+ 
+ Pour désarchiver les fichiers, vous pouvez utiliser le gestionnaire d'archives ou exécuter la commande ci-dessous. Les fichiers doivent être placés dans le répertoire du serveur Web
+
+   ```bash
+tar -xzf "fichiers.tar.gz" -C / var / www / html
+
+où:
+
+     / var / www / html est un répertoire de serveur Web.
+
 
 ### Step 2. Set required permissions
 
+    Etape 2. Régler les autorisations requises
+
 The files should be owned by a web-server user and have correct permissions. Please set required permissions by following this instruction: [www.espocrm.com/documentation/administration/server-configuration/#user-content-required-permissions-for-unix-based-systems](https://www.espocrm.com/documentation/administration/server-configuration/#user-content-required-permissions-for-unix-based-systems).
 
+Les fichiers doivent appartenir à un utilisateur de serveur Web et avoir les permissions correctes. Veuillez régler définir les autorisations requises en suivant cette instruction s’il vous plait. www.espocrm.com/documentation/administration/server-configuration/#user-content-required-permissions-for-unix-based-systems.
+
+
+

+    Étape 3. ımporter la base de données dump
 
 :
 
 ```bash
 mysql --user=YOUR_DATABASE_USER --password=YOUR_DATABASE_PASSWORD YOUR_DATABASE_NAME < db.sql
+
+La base de données dump doit être importée dans la même base de données avec les mêmes informations d'utilisateur, Sinon la confirmation doit être effectuée dans le fichier de configuration ESPOCRM_DIRECTORY / data / config.php. Pour importer votre base de données à partir de la décharge , exécutez la commande ci-dessous dans un terminal
 ```
+mysql --user=YOUR_DATABASE_USER --password=YOUR_DATABASE_PASSWORD YOUR_DATABASE_NAME < db.sql
+
 
      
+    Étape 4. Vérifier / configurer crontab
 
 
 
@@ -94,6 +152,20 @@ If you have to make any changes, use this command:
 
 ```bash
 sudo crontab -l -u www-data
-```
 
-More details about configuring crontab for EspoCRM is described here [www.espocrm.com/documentation/administration/server-configuration/#user-content-setup-a-crontab](https://www.espocrm.com/documentation/administration/server-configuration/#user-content-setup-a-crontab).
+Vérifiez si votre crontab est configuré d’une façon convenable.Exécutez la commande ci-dessous et vérifiez si un chemin vers EspoCRM est correct:
+
+sudo crontab -l -u www-data
+
+où:
+
+    . www-data est votre utilisateur de serveur Web.
+
+Si vous devez faire un changement quelconque utilisez cette commande:
+
+sudo crontab -l -u www-data
+
+
+`More details about configuring crontab for EspoCRM is described here [www.espocrm.com/documentation/administration/server-configuration/#user-content-setup-a-crontab](https://www.espocrm.com/documentation/administration/server-configuration/#user-content-setup-a-crontab).
+
+Plus de détails sur la configuration de crontab pour EspoCRM est décrit ici www.espocrm.com/documentation/administration/server-configuration/#user-content-setup-a-crontab